s-sql/tests/tests.lisp

```diff
@@ -10,8 +10,24 @@
 
 (fiveam:in-suite :s-sql)
 
+(defun prompt-connection-to-s-sql-db-spec (param-lst)
+  "Takes the 6 item parameter list from prompt-connection and restates it for pomo:with-connection. Note that cl-postgres does not provide the pooled connection - that is only in postmodern - so that parameter is not passed."
+  (when (and (listp param-lst)
+             (= 6 (length param-lst)))
+    (let ((db (pop param-lst))
+          (user (pop param-lst))
+          (password (pop param-lst))
+          (host (pop param-lst))
+          (port (pop param-lst))
+          (use-ssl (pop param-lst)))
+      (list db user password host :port port :use-ssl use-ssl))))
+
+(defvar *s-sql-db-spec* nil
+  "A list of connection parameters to use when running the tests.  The order is: database name, user, password, hostname, port and use-ssl.")
```

Comment:
Semantically, `'()` would be clearer than `nil` in this context - `nil` denotes a boolean whereas `'()` denotes an empty list.

Reply:
Agreed